language/documentation/hackathon-tutorial/README.md

```diff
@@ -0,0 +1,599 @@
+# move-hackathon-tutorial
+Move tutorial for the hackathon happening on Dec 7-8
+
+### Step 0
+## TODO: use binaries instead
```

Comment:
Nit: use `> TODO bla` in MD to mark todos. Sections here make things rather unreadable.

Also why to use binaries? If you do, those should be not part of the ckecked-in tutorial, but rather specific instructions for tutorials.